backend/src/main/java/org/globe42/web/persons/PersonDTO.java

```diff
@@ -149,6 +151,8 @@ public HealthCareCoverage getHealthCareCoverage() {
         return healthCareCoverage;
     }
 
+    public String getHealthInsurance() { return healthInsurance; }
```

Comment:
Please, rebase on master, and use the same format for this getter (and the other ones) as for all the other ones (instructions of the method on a separate line):

```
public String getHealthInsurance() { 
    return healthInsurance; 
}
```

I guess you would like the code to be more concise, but I suggested using Kotlin :-) 

And to me that makes it inconsistent, and less readable.